app/workers/publishing_api_services_and_information_worker.rb

```diff
@@ -0,0 +1,29 @@
+class PublishingApiServicesAndInformationWorker < PublishingApiWorker
+  def perform(organisation_id)
+    organisation = Organisation.find(organisation_id)
+    if organisation.has_services_and_information_link?
+      payload = PublishingApi::ServicesAndInformationPresenter.new(organisation)
+      # Get the content_id for the content item, or set a new one if it's
+      # a new content item
+      content_id = content_id_for_base_path(
+        payload.content[:base_path]
+      ) || SecureRandom.uuid
+
+      Whitehall.publishing_api_v2_client.put_content(
```

Comment:
I think you can use [`#send_item`](https://github.com/alphagov/whitehall/blob/master/app/workers/publishing_api_worker.rb#L27) on the base class now.